frontend/src/api/invest.ts

```diff
@@ -30,23 +30,43 @@ export async function getLoansToInvest(cb: (loans: LoanToInvestT[]) => void): Pr
   cb(loansToInvest);
 }
 
-export async function getLoanToInvest(shortId) {
+export async function getLoanToInvest(shortId: string, cb: (loan: LoanToInvestT) => void) {
```

Comment:
Again, why the callback? Just return the value, it's an async function.

Reply:
right!